packages/domain/lib/src/utils/nullable_extensions.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:collection/collection.dart';
+
+extension EmptyStringIfNull on String? {
+  String get stringOrEmpty => this ?? '';
+}
+
+extension GetFirstIOrDefaultValue on List<String>? {
+  int get firstToInt {
+    final value = this?.firstOrNull ?? '0';
+    return int.parse(value);
+  }
```

Comment:
imho, it's better to return null instead of 0. but it's up to u

Reply:
I just try to avoid '??', I feel peace when I can avoid it, so I had some free time to of this =)